docker_test.go

```diff
@@ -697,6 +762,16 @@ func TestPostgres(t *testing.T) {
 	require.Equal(t, myEvent.count, "1")
 }
 
+// Verify Event in POSTGRES
+func TestReportingService(t *testing.T) {
+	var myEvent Event
+	require.Eventually(t, func() bool {
+		eventSql := "select count (*) from metrics"
+		rs_db.QueryRow(eventSql).Scan(&myEvent.count)
+		return myEvent.count == "11"
```

Comment:
* The `myEvent.count == "11"` seems overly specific and easy to break. 

* We don't check for the content of those metrics here. Thus, if we simply insert empty or incorrect rows the test won't detect it.

* Is it possible to query the reporting service for the expected data without looking directly into the database. APIs are generally more stable than underlying db schema.